sdk/php/src/ValueObject/Type.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dagger\ValueObject;
+
+use Dagger\Client\IdAble;
+use Dagger\TypeDefKind;
+use ReflectionClass;
+use ReflectionNamedType;
+use ReflectionType;
+use RuntimeException;
+
+//@TODO support union/intersection types
```

Comment:
Dagger doesn't support [union types](https://graphql.org/learn/schema/#union-types). If this is referring to unions in PHP only, I'd be curious to know how that translates. In Python and TypeScript, the only union is with nulls (`<type> | None`). Could also be unions with similar types that map to the same thing when serialized (e.g., str and bytes, both serialized as `String` in the API).